src/sentry/static/sentry/app/views/settings/account/avatar.jsx

```diff
@@ -1,76 +1,74 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import {connect} from 'react-redux';
+
 import styled from 'react-emotion';
-import createReactClass from 'create-react-class';
 
 import Panel from '../components/panel';
 import PanelBody from '../components/panelBody';
 import PanelHeader from '../components/panelHeader';
 import AlertActions from '../../../actions/alertActions';
-import ApiMixin from '../../../mixins/apiMixin';
 import AvatarCropper from '../../../components/avatarCropper';
 import RadioGroup from '../components/forms/radioGroup';
 import LoadingError from '../../../components/loadingError';
 import LoadingIndicator from '../../../components/loadingIndicator';
 import {t} from '../../../locale';
 
-const AvatarSettings = createReactClass({
-  displayName: 'AvatarSettings',
-
-  propTypes: {
+class AvatarSettings extends React.Component {
```

Comment:
aside, this kind of behavior is what AsyncView was really intended to solve: a component which needs to fetch data to render